doc/source/howtos/upgrade_guide.rst

```diff
@@ -117,3 +120,41 @@ To upgrade the cluster, follow these steps for each node:
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
+It is important that when rejoining, the node should synchronize using
+:term:`IST`.
+
+When performing any upgrade (major or minor), :term:`SST` could
+be initiated by the joiner node after the upgrade if the server
+was offline for some time. After :term:`SST` completes, the data
+directory structure needs to be upgraded (using mysql_upgrade)
+once more time to ensure compatibility with the newer version
+of binaries.
+
+   .. note:: In case of :term:`SST` synchronization,  messages log
```

Comment:
s/synchronization,  messages/synchronization, the messages/

Reply:
Is the "messages log" the "error log"? If so, should it say "error log" instead? @kbauskar ?

Reply:
@RoelVdP "it is error log."

Reply:
Okay, let's use "error log" then as that is what people are used to. (@fiowro this applies to all documentation.)